Tests/iaas/entropy/entropy-check.py

```diff
@@ -428,7 +428,7 @@ def main(argv):
     try:
         logger.debug(f"Connecting to cloud '{cloud}'")
         with openstack.connect(cloud=cloud, timeout=32) as conn:
-            all_images = conn.list_images()
+            all_images = list(filter(lambda x: x.visibility == 'public' or x.visibility == 'community', conn.list_images()))
```

Comment:
AFAICT, the image-metadata check uses server-side filtering, and it only checks public images. Shall we change that as well?

Reply:
Three points:
1. Having consistency between the two checks would always be preferable.
2. I did not see an obvious way to do server side filtering, but I may not have looked deeply enough. It's always preferable.
3. I chose `public` and `community` because these are controlled by the cloud operator.

Reply:
So, let's change both tests, then?